src/main/scala/uncore/converters/Tilelink.scala

```diff
@@ -398,6 +413,7 @@ class TileLinkIONarrower(innerTLId: String, outerTLId: String)
   val shrink = iacq.a_type === Acquire.getBlockType
   val smallput = iacq.a_type === Acquire.putType
   val smallget = iacq.a_type === Acquire.getType
+  val atomic = iacq.a_type === Acquire.putAtomicType
```

Comment:
do you need assertions on the outer size similar to the ones on smallput and smallget below (line 446)?